### PR TITLE
README.md: "Download/clone the source code"

### PR DESCRIPTION
**Pull request by isghe**
Make it clear on how to build from sources
(issue: https://github.com/ethereumproject/go-ethereum/issues/280 )

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -29,7 +29,11 @@ If your heart is set on the bleeding edge, install from source. However, please
 #### Dependencies
 Building geth requires both Go >=1.8 and a C compiler.
 
+#### Download/clone the source code
+`$ go get -v github.com/ethereumproject/go-ethereum/...`
+
 #### Installing command executables
+`$ cd $GOPATH/src/github.com/ethereumproject`
```
Comment (whilei) [inline README.md]: This step is unnecessary

File: README.md [hunk 2]
```diff
@@ -29,7 +29,11 @@ If your heart is set on the bleeding edge, install from source. However, please
 #### Dependencies
 Building geth requires both Go >=1.8 and a C compiler.
 
+#### Download/clone the source code
```
Comment (whilei) [inline README.md]: Let's reword this to `Get source and dependencies`, since it's not `clone`ing (in the __git__ sense) and it'll grab more than just the source (the `/...` tells go to _get_ all the package's required dependencies as well).

**Comment by whilei** [comment]
@isghe Thanks for pitching in! Glad to have you here.

**Comment by whilei** [comment]
👍  Thanks again @isghe

**Comment by isghe** [comment]
Thanks to you @whilei :-)
